downloads/1.2.x-release-notes/1.2.41.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 1.2.41
+permalink: /downloads/1.2.x-release-notes/1.2.41/
+active: 1.2.41
+redirect_from:
+    - /downloads/1.2.x-release-notes/
+---
+
+### Overview of jBallerina 1.2.41
+
+The jBallerina 1.2.41 patch release will fix a bug in the http standard library. Where the bug is "inconsistencies in the multipart MIME encoder".
```

Comment:
```suggestion
The jBallerina 1.2.41 patch release will fix a bug in the http standard library. Where the bug is "inconsistencies in the multipart MIME encoder".
```
Please change similar to https://ballerina.io/downloads/1.2.x-release-notes/1.2.37. Please fix the same in https://ballerina.io/downloads/1.2.x-release-notes/1.2.40 as well.